src/renderer/apiRefresh.ts

```diff
@@ -0,0 +1,45 @@
+// ==LICENSE-BEGIN==
+// Copyright 2017 European Digital Reading Lab. All rights reserved.
+// Licensed to the Readium Foundation under one or more contributor license agreements.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file exposed on Github (readium) in the project repository.
+// ==LICENSE-END==
+
+import { TApiMethodName } from "readium-desktop/main/api/api.type";
+import { TMethodApi, TModuleApi } from "readium-desktop/main/di";
+import { diRendererGet } from "readium-desktop/renderer/di";
+import { ApiLastSuccess } from "readium-desktop/renderer/redux/states/api";
+
+/**
+ * subscribe to redux to automaticaly execute callback when any path in parameter is called on IPC
+ *
+ * don't forget to unsubscribe with the return value of function
+ */
+export function apiRefresh(pathArrayToRefresh: TApiMethodName[], cb: () => void | Promise<void>) {
```

Comment:
Just to improve code legibility / readability, I would add the return type `Unsubscribe` explicitly (rather than relying on implicit type inference in IDEs like Visual Studio Code)

Reply:
3553a8a